### PR TITLE
fix: Add missing `pages-functions-wasm-app` deps in `package-lock.json`

### PR DESCRIPTION
**Pull request by CarmenPopoviciu**
What this PR solves / how to test:
Merging https://github.com/cloudflare/wrangler2/pull/2554 seems to have broken CI
https://github.com/cloudflare/wrangler2/actions/runs/3986649825/jobs/6835473536, so this PR attempts to fix that

Associated docs issues/PR:

- [insert associated docs issue(s)/PR(s)]

Author has included the following, where applicable:

- [ ] Tests
- [ ] Changeset

Reviewer has performed the following, where applicable:

- [ ] Checked for inclusion of relevant tests
- [ ] Checked for inclusion of a relevant changeset
- [ ] Checked for creation of associated docs updates
- [ ] Manually pulled down the changes and spot-tested

Fixes # [insert issue number].

### CODE REVIEW

**Review by GregBrimble — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -28242,7 +28268,7 @@
 		},
 		"packages/pages-shared": {
 			"name": "@cloudflare/pages-shared",
-			"version": "0.0.14-prerelease-1",
+			"version": "0.0.13",
```
Comment (petebacondarwin) [inline package-lock.json:28271]: What's this change about?

Comment (CarmenPopoviciu) [reply package-lock.json:28271]: hmmm...I don't think that should be there. @GregBrimble is this smth we should address in a separate PR?

**Comment by CarmenPopoviciu** [comment]
@petebacondarwin any idea why this wasn't caught in my original PR? both `devtools.yml` and `pullrequests.yml` seem to run the same `npm ci` command

File: package-lock.json (hunk 1 above)
Comment (CarmenPopoviciu) [reply package-lock.json]: oh! ok then...changing back

**Pull request merged**